modeling-cmds/src/def_enum.rs

```diff
@@ -695,6 +695,20 @@ define_modeling_cmd_enum! {
             pub color: Color,
         }
 
+        /// Set the color of the tool lines for the scene.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct SetCurrentToolProperties {
+            /// The color to set the tool line to.
+            pub color: Option<Color>,
+        }
+
+        /// Set the default color used when a specific color isn't used.
```

Comment:
This comment shouldn't mention colour specifically anymore (same for the above)

Reply:
Done